docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml

```diff
@@ -340,6 +340,12 @@ ecl deploy roxie - --name=FindPersonService
                   <entry>The published query name</entry>
                 </row>
 
+                <row>
+                  <entry>--protect</entry>
+
+                  <entry>Protect the workunit from deletion </entry>
```

Comment:
I don't see an extra space on the other entries... was this deliberate?

Reply:
@afishbeck Yes, it is deliberate and the same as other rows that have text in two columns.